task/crda-analysis/0.1/crda-analysis.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+
+metadata:
+  name: crda-analysis
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: 0.19.0
+    tekton.dev/tags: Security, Vulnenrability, CVE
+    tekton.dev/displayName: CRDA Analysis
+
+spec:
+  description: >-
+    The CRDA Analysis task is an interface between Tekton and Red Hat CodeReady Dependency Analytics platform.
+    It provides vulnerability and compliance analysis for your Python applications dependencies, along with recommendations to address security vulnerabilities and licensing issues.
+
+  workspaces:
+    - name: output
+      description: Volume backing this workspace is used for input/output of the task.
+
+  params:
+    - name: manifest-file-path
+      description: Path of target manifest file to perform analysis.
+      default: $(workspaces.output.path)/requirements.txt
+
+    - name: output-file-path
+      description: Path of the file to save analysis report.
+      default: $(workspaces.output.path)/crda-analysis-report.json
+
+    - name: pkg-installation-directory-path
+      description: Path of a directory in workspace, where dependencies are installed.
+      default: $(workspaces.output.path)/site-package
```

Comment:
instead of setting `$(workspaces.output.path)` in the default probably you can set the `workingDir` as 
`workingDir: $(workspaces.output.path)` in the step

Reply:
Yes @vinamra28, that is what I would like to do, but currently issue is that [CRDA-CLI](https://github.com/fabric8-analytics/cli-tools/releases) used in this task requires absolute path to files, that is why I need to add $(workspaces.output.path) in path. 
Request to support relative path in CRDA-CLI is raised and soon it will be supported. Once this is done we no longer need to provide $(workspaces.output.path).

Reply:
This is done.